src/app/components/ReduxForm/constants.js

```diff
@@ -0,0 +1,40 @@
+import i18 from 'i18next';
+
+import { required, phoneNumber, justCharacters } from './utils';
+import styles from './styles.module.scss';
+
+export const TEXTAREA_COLS = 10;
+export const TEXTAREA_ROWS = 30;
```

Comment:
extract this constants into a constants.js file inside src/constants/yourFile.js and then import these values into this file and into InputForm/Components/SurveyField/constants.js file too